app/src/main/java/com/morihacky/android/rxjava/RotationPersistWorkerFragment.java

```diff
@@ -66,6 +68,8 @@ public Integer call(Long aLong) {
         //_intsObservable = _intsObservable.share();
         storedIntsObservable = intsObservable.replay();
 
+        storedIntsSubscription = storedIntsObservable.connect();
```

Comment:
help me understand again why `share` a.k.a `publish().refcount()` is better than `.replay().connect()` again? you mentioned on Twitter, but what difference would it make if i anyway have my subscription being created here?